content/post/2022-11-08-sponsor.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Contribuer autrement"
+date: 2022-11--8T14:55:31+02:00
```

Comment:
I think there's a typo on the date format, day should be `08` instead of `-8`, Hugo is very picky with that (I once spent 2 hours to figure out why my post wasn't published)

Reply:
Nice catch. Thanks!